sources/packages/web/src/views/aest/student/StudentRestrictions.vue

```diff
@@ -10,6 +10,7 @@
             ><v-btn
               @click="addStudentRestriction"
               class="float-right primary-btn-background"
+              data-cy="addRestriction"
```

Comment:
I noticed that we have been using the suffix _Button_ for other cases. Maybe it would be good to keep using it consistently.

Reply:
Added suffix for buttons